### DISCUSSION TITLE
Unable to use 5.0.0-rc.1 after upgrading from 4.0.0

### DISCUSSION THREAD

**Discussion by StephenHodgson** [wixtoolset/issues#8037] [Questions] [answered]
I had previously installed 4.0.4, and had a pretty small project setup.

I wanted to use the new features for Built-in file harvesting introduced in 5.0.0-rc.1.

I upgraded wix using dotnet tool pwsh

When I opened my project and tried to use the new `<Files Include="path/to/my/files/**" />` schema, I got the following:

```
Severity	Code	Description	Line	Suppression State	Details
Warning		The element 'Package' in namespace 'http://wixtoolset.org/schemas/v4/wxs' has invalid child element 'Files' in namespace 'http://wixtoolset.org/schemas/v4/wxs'. List of possible elements expected: 'AdminExecuteSequence, AdminUISequence, AdvertiseExecuteSequence, AppId, Binary, ComplianceCheck, Component, ComponentGroup, CustomAction, CustomActionRef, CustomTable, CustomTableRef, Directory, DirectoryRef, EmbeddedChainer, EmbeddedChainerRef, EnsureTable, Feature, FeatureRef, FeatureGroupRef, Icon, InstallExecuteSequence, InstallUISequence, InstanceTransforms, Launch, MajorUpgrade, Media, MediaTemplate, PackageCertificates, PatchCertificates, Property, PropertyRef, Requires, SetDirectory, SetProperty, SFPCatalog, SoftwareTag, StandardDirectory, SummaryInformation, SymbolPath, UI, UIRef, Upgrade, WixVariable' in namespace 'http://wixtoolset.org/schemas/v4/wxs' as well as any element in namespace '##other'.	8		
Severity	Code	Description	Line	Suppression State	Details
Error	WIX0005	The Package element contains an unexpected child element 'Files'.	8		
```

I then tried to update the wixproj manually from 4.0.4 -> 5.0.0-rc.1

```
<Project Sdk="WixToolset.Sdk/5.0.0-rc.1">
</Project>
```

which then caused the project to fail to load. 

```
./NativeWindowsInstaller.wixproj : error  : The project file cannot be opened by the project system, because it is missing some critical imports or the referenced SDK cannot be found.
```

Did I miss a step in upgrading? I know it is still in pre-prelease but this specific feature is worth testing it out early.

**Comment by robmen** [comment, marked as answer]
HeatWave does not yet support WiX v5. You'll want to stick to the command-line (`wix build` or `dotnet build` or MSBuild`) and your favorite text editor like the [earliest episodes of the Deployment Dojo](https://robmensching.com/deployment-dojo/episodes/s1/)... until there is a HeatWave update.

And please do try out the newest features. :)

**Reply by StephenHodgson** [reply]
Understood! Yeah I had a feeling that I was a little too early to the party 😂

**Comment by rowbot38** [comment]
Still a problem with rc2
